resources/prosody-plugins/mod_visitors_component.lua

```diff
@@ -397,20 +397,6 @@ process_host_module(muc_domain_prefix..'.'..muc_domain_base, function(host_modul
             return false;
         end
 
-        -- let's forward to every moderator, this is so they now that this moderator
-        -- took action and they can update UI, as this msg was initially a group chat but we are
-        -- sending it now as provide chat, let's change the type
-        stanza.attr.type = 'chat'; -- it is safe as we are not using this stanza instance anymore
```

Comment:
This is not used anymore? Isn't this supposed to notify the other moderators to remove the notification for approval? How it is working now?

Reply:
the visitor prosody lowers the hand of the promoted participant before leaving.